suite-common/token-definitions/src/index.ts

```diff
@@ -1,7 +1,8 @@
-export * from './types';
-export * from './utils';
-export * from './constants';
+export * from './tokenDefinitionsUtils';
+export * from './tokenDefinitionsConstants';
 export * from './tokenDefinitionsSelectors';
 export * from './tokenDefinitionsReducer';
 export * from './tokenDefinitionsThunks';
 export * from './tokenDefinitionsMiddleware';
+export * from './tokenDefinitionsTypes';
+export * from './tokenDefinitionsTypes';
```

Comment:
Do not rename it. Types is twice here

Reply:
Missed that but as explained above I think that renaming them is good to keep structure for redux stuff the same.